src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There are exceptions to all the rules but in general I would prefer the builds files be 1:1 with the csproj's. Only if there is some strange coupling between the different csproj's should we put them together. In some cases if there are helper csproj's then they don't even really need to be in builds file they should just be project references and pulled in from the root test project.